roles/replace_node/meta/main.yml

```diff
@@ -0,0 +1,57 @@
+galaxy_info:
+  author: your name
```

Comment:
Update this?

Reply:
Yep, removeing that file. We do not need it now.

Reply:
Ack! Done.